apis/logging/v1/output_types.go

```diff
@@ -276,6 +276,21 @@ type Splunk struct {
 	// Should be a valid JSON object
 	// +optional
 	Fields []string `json:"fields,omitempty"`
+
+	// IndexKey is a meta-data key field to use to send events to.
+	// For example: 'IndexKey: kubernetes.namespace_name` will use the kubernetes
```

Comment:
Ideally, I think we should restrict these to a limited field set (e.g. kubernetes.namespace_name, kubernetes.pod_name) but I do not find anyplace where we do verification or additionally make that limitation so I guess it is consistent